remappings.txt

```diff
@@ -0,0 +1,3 @@
+weird-erc20/=lib/solmate/lib/weird-erc20/src/
```

Comment:
Is this one required, or does it work without it?

Reply:
Running `forge remappings` gives me these 3 lines. Seems like it's working fine without the first line (`weird-erc20`). Line removed. 👍